### PR TITLE
Adds a huge message to sec borgs telling them not to be shitheads.

### PR DESCRIPTION
**Pull request by theOperand**
self-explanatory.

### CODE REVIEW

**Comment by optimumtact** [comment]
Worth a try I guess

**Comment by Cebasiz** [comment]
follow the durn laws

**Comment by Fox-McCloud** [comment]
Technically speaking, doesn't this mean borgs *can't* ever stun baton a person? I mean, stun batons apply an electrical shock that hurts like hell and debatably causes harm---same with tasers.

**Comment by Cebasiz** [comment]
harm is only HP harm

see: mediborgs can inject people, can do surgery etc.

otherwise it would quickly get ridiculous with humans holding other humans hostage with emotional harm by calling them fat et cetera.

**Comment by Miauw62** [comment]
yeah, as long as youre on help batons are not harmful. pain is not harm.

**Comment by Cheridan** [comment]
remove

**Comment by Miauw62** [comment]
the problem with removing secborgs is that they are pretty vital for traitor/malf ais.

**Comment by AlexanderUlanH** [comment]
I liked the idea of the detective borg.  If that makes malf AI too weak, they could always get a strong hacked module.

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 1]
```diff
@@ -198,7 +198,7 @@
 			icon_state = "secborg"
 			animation_length = 28
 			modtype = "Sec"
-			//speed = -1 Secborgs have nerfed tasers now, so the speed boost is not necessary
+			src << "<span class='userdanger'>While you have picked the security module, you still have to follow your laws, NOT Space Law. For Asimov, this means you must follow criminals' orders unless there is a law 1 reason not to.</span>"
```
Comment (Dorsisdwarf) [inline code/modules/mob/living/silicon/robot/robot.dm]: "Security borgs are still bound by Asimov not Space Law, with all laws and the commands of any AI you are synced to coming first. 
While not required to enforce Space Law, you may do so if the act of doing so does not violate a law or command from any synced AI.
if(asimov)
Arresting a non-harmful human for a non-harmful crime and ignoring a request from them to be let go is a violation of ASIMOV laws.

**Comment by SconesC** [comment]
This is really dumb and pretty much a big OOC message

If borgs are too dumb to listen to common sense they get jobbanned

**Comment by optimumtact** [comment]
Why is it dumb? Lots of team antags and other types of modes have messages just like this when you get converted or flashed etc.

**Pull request merged**